### ISSUE TITLE
Feature Request: Login Option in webnovel.com 

### ISSUE THREAD

**Issue by budikesuma** [lncrawl/lightnovel-crawler#1748]
Hello dev,
@dipu-bd 

It would be desirable to apply the ability of the login option to scrape the chapters that have been unlocked on our account.
Thanks...

**Comment by dipu-bd** [comment]
If you know what to modify in the code to make it work, you can just send a Pull Request with your changes

**Comment by dipu-bd** [comment]
Kept open, with the dependency recorded so it is clear what this is waiting on.

Logging into a source means storing the credentials somewhere, and there is no credential store yet — that is #3012, which notes the internal functionality already exists and only the frontend and storage are missing. This request lands as soon as that does.

**Comment by dipu-bd** [comment]
Closing as a duplicate of #3012, which now tracks credential storage as one thing.

This is not a dismissal — logging in here is wanted, and the crawler side of it already exists. What is missing is somewhere safe to keep the account details, and that gap is the same for every source, so it is being solved once rather than per site. Follow #3012.
